source.py

```diff
@@ -0,0 +1,11 @@
+import cv2
```

Comment:
Change file name to edit_image.py and change file path to feature/opencv/ folder. ( check docker_container ) branch